### PR TITLE
Expand discussion of strengths and weaknesses of inference methods.

### PR DESCRIPTION
**Pull request by RyanGutenkunst**
Added a paragraph in discussion to compare inference methods a tad bit more, while still dodging a full comparison. Happy for feedback on other aspects we should mention regarding tool choice.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -614,9 +614,20 @@ \section*{Discussion}
...
-with the CEU history but tends to overfit the YRI history.  The results for
+with the CEU history but tends to overfit the YRI history.
+The results for
 the two-population \textit{D.~melanogaster} model (Figure~\ref{fig:two_popn_fly})
 are generally similar.
+In these comparisons, \fastsimcoal and \dadi perform almost identically, which is expected because they fit the same models to the same summaries of the data, differing only in how they calculate model expectations and optimize parameters.
+
+All methods for inferring demographic history have strengths and weaknesses (as recently reviewed by \citet{beichman2018review}.)
+We compared inferences from simulated whole genome data, but many factors affect choice of methodology.
+Markovian coalescent methods (\MSMC and \smcpp) require long contiguous stretches of sequence data.
```
Comment (grahamgower) [inline popSimManu.tex:625]: Actually, 100kb contigs can be enough for psmc.
https://www.biorxiv.org/content/10.1101/382036v1

File: response-to-reviewers.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ This will definitely need to be removed before we send the actual response!
 > 3) We felt that the discussion would benefit from a discussion of the details not yet incorporated into the platform, e.g. sex-specific recombination, gene conversion, assembly error, structural variants etc. Such an addition might spur future development, and also give the authors an opportunity to discuss general ways forward on these issues. 
 > 
 > 4) We would be interested to see more of a discussion of the strengths and weakness of the various demographic inference methods. 
-> 
+
+We have added an additional paragraph to the discussion exploring some of the additional factors that may affect choice of inference method, beyond results like those presented here. These include data required, type of model, and software engineering. A comprehensive investigation of these issues is beyond the scope of the current manuscript, with focuses on the resource itself, rather than its application.
```
Comment (jeromekelleher) [inline response-to-reviewers.md]: s/with/which

**Review by jeromekelleher — COMMENTED** [review]
LGTM. I spotted one typo.

File: popSimManu.tex (hunk 1 above)
Comment (RyanGutenkunst) [reply popSimManu.tex:625]: Good point, I wasn't aware of your work on this. But I think we can still call 100 kb contigs "long", certainly relative to RADseq.

**Review by petrelharp — COMMENTED** [review]
I like it, looks good.

**Comment by jeromekelleher** [comment]
@andrewkern, can you merge if you're happy please?

**Comment by andrewkern** [comment]
sorry for being slow here! i have my head down with teaching

**Comment by RyanGutenkunst** [comment]
No worries Andy. Thanks!

**Pull request merged**